.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 {
+    "editor.formatOnSave": false, // conflicts with ESLint format
```

Comment:
If other devs need to update/revert their IDE settings, please let us know on slack!

Reply:
OK, let me discuss about it later. For now, that line has been reverted.